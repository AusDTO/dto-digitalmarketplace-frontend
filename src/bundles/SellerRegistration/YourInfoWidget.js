import React from 'react'
import { Provider } from 'react-redux'
import RegisterComponent from '../../RegisterComponent'

import createStore from './redux/create'
import YourInfoForm from './components/YourInfoForm'

export const YourInfoWidget = (props) => {
  const store = createStore(props)
  return (
    <Provider store={store}>
      <YourInfoForm />
    </Provider>
  )
}

export default new RegisterComponent({ yourinfo: YourInfoWidget })
