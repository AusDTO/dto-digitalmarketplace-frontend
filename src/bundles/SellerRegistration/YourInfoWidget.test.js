// FIXME This is a workaround.
// @see https://github.com/facebook/react/issues/7386
jest.mock('react-dom');

import React from 'react';
import { YourInfoWidget } from './YourInfoWidget';
import renderer from 'react-test-renderer';

test('YourInfoWidget renders without errors', () => {
  const component = renderer.create(
    <YourInfoWidget />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})