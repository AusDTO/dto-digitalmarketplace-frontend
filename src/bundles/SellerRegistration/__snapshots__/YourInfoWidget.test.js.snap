exports[`test YourInfoWidget renders without errors 1`] = `
<div
  className="layout">
  <header>
    <h1>
      Your Information
    </h1>
  </header>
  <article
    className="content-main"
    role="main">
    <form
      style={
        Object {
          "display": "none",
        }
      }
      tabIndex="-1" />
    <form
      action={undefined}
      id="yourinfo"
      method="post"
      onReset={[Function]}
      onSubmit={[Function]}>
      <div
        className="field">
        <label
          className="question-heading"
          htmlFor="representative">
          Company Representative
        </label>
        <input
          aria-describedby={undefined}
          className=""
          id="representative"
          name="representative"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyPress={[Function]}
          type="text"
          value="" />
      </div>
      <div
        className="field">
        <label
          className="question-heading"
          htmlFor="name">
          Company Name
        </label>
        <input
          aria-describedby={undefined}
          className=""
          id="name"
          name="name"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyPress={[Function]}
          type="text"
          value="" />
      </div>
      <div
        className="field">
        <label
          className="question-heading"
          htmlFor="abn">
          ABN
        </label>
        <input
          aria-describedby={undefined}
          className=""
          id="abn"
          name="abn"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyPress={[Function]}
          type="text"
          value="" />
      </div>
      <div
        className="field">
        <label
          className="question-heading"
          htmlFor="phone">
          Phone
        </label>
        <input
          aria-describedby={undefined}
          className=""
          id="phone"
          name="phone"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyPress={[Function]}
          type="text"
          value="" />
      </div>
      <div
        className="field">
        <label
          className="question-heading"
          htmlFor="email">
          Email
        </label>
        <input
          aria-describedby={undefined}
          className=""
          id="email"
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyPress={[Function]}
          type="text"
          value="" />
      </div>
      <input
        onClick={[Function]}
        role="button"
        type="submit"
        value="Save & Continue" />
    </form>
  </article>
</div>
`;
